### DISCUSSION TITLE
Auto-delete spam

### DISCUSSION THREAD

**Discussion by dlh2** [friedPotat0/Spam-Scores#42] [Ideas]
To sum it up,

Thanks to the addon we have new parts for filtering messages:
![image](https://user-images.githubusercontent.com/14979623/137090039-4fad52c6-fe79-420e-ba56-c84b66a65a3e.png)

But as email clients they don't have anything for:
![image](https://user-images.githubusercontent.com/14979623/137090190-2463f8be-dea9-4e0f-b6cd-eacaf2593332.png)

If score > -4 then delete.

The reason behind this, is that there are mail servers who don't offer anything for deleting the worst Spam, instead they just give the score headers and they simply think "Ohh you can delete them if you find our tag [SPAM]" but of course they have a threshold of accuracy so that's one of the main reasons I was really interested in this addon.

I find it really strange that there's no email client which offers this.

Note: Yeah the images are in Spanish.

**Comment by friedPotat0** [comment]
For people not having the possibility to adjust the threshold where the server moves certain mails to the spam folder it would definitely be an improvement. I think we should be careful about directly deleting such mails. Maybe we should automatically move them to the junk folder instead of deleting them.

**Reply by dlh2** [reply]
Indeed, the idea is that we can somehow configure it, by default it should be disabled.
